cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/kafkaassigner/KafkaAssignerEvenRackAwareGoal.java

```diff
@@ -112,11 +112,10 @@ public boolean optimize(ClusterModel clusterModel, Set<Goal> optimizedGoals, Set
     Map<String, Integer> replicationFactorByTopic = clusterModel.replicationFactorByTopic();
```

Comment:
We should delete `replicationFactorByTopic`, because it is not used after the patch.